clients/driver/handler.js

```diff
@@ -1,25 +1,26 @@
 'use strict';
 
 
-const eventEmitter = require('../eventEmitter');
+const { io } =  require('socket.io-client');
+const socket =  io('http://localhost:3002/caps');
```

Comment:
Also, be careful with creating sockets in both the index and handler files; you did a good job passing the socket to the vendor handler as a function argument.